doc/how_to/index.md

```diff
@@ -2,6 +2,18 @@
 
 Lumen's How to Guides provide step by step recipes for solving essential problems and tasks. They are more advanced than the Getting Started material and assume some knowledge of how Lumen works.
 
+## Configuring Lumen AI
+
+::::{grid} 1 2 2 3
+:gutter: 1 1 1 2
+
+:::{grid-item-card} {octicon}`gear;2.5em;sd-mr-1` Configuring Lumen AI
```

Comment:
This doesn't match how the others are done. The others add one item per how-to guide and unless we think there will be a ton of guides we should do the same here.

Reply:
So you're suggesting a `lumen_agents`, `lumen_analyses`, `lumen_tools` `lumen_prompts` directory?

<img width="174" alt="image" src="https://github.com/user-attachments/assets/1900d251-7f9c-4f88-b2a7-2ecb2478fa3b" />

Reply:
No I'm suggesting listing out each of guides, i.e. create a card for `ai_config/custom_agents`, `ai_config/custom_analyses` ...